doc/content/getting-started/migrating/device-json.md

```diff
@@ -45,6 +45,26 @@ Multiple end devices can also be contained in a single `devices.json` file like
 ]
 ```
 
+***Important observation***
+
+The format described above works for the console, when using the cli a JSON stream is expected. One of the possible ways of doing it in the cli would be:
+```bash
+cat end_device_1.json end_device_2.json | ttn-lw-cli end-device create 
+```
+Another way would be to have the contents already concatenated in a single file and use that.
```

Comment:
I would say this is the preferred way

Reply:
Indeed.

The section was removed in favour of a more simple explanation